docs/releases/3.0.txt

```diff
@@ -66,6 +66,18 @@ async code before, this may trigger if you were doing it incorrectly. If you
 see a ``SynchronousOnlyOperation`` error, then closely examine your code and
 move any database operations to be in a synchronous child thread.
 
+Cross-DB JSONField
```

Comment:
Perhaps "JSONField implementation supported by all built-in database backends"?

Reply:
Isn't that a bit too long? How about "JSONField for all supported database backends"?